tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -23,7 +23,8 @@
     "\n",
     "### Course structure\n",
     "\n",
-    "The tutorial is thought across 12 jupyter notebook:\n",
+    "The tutorial is thought across 13 jupyter notebooks:\n",
```

Comment:
`a series of` instead of `13`? This way we don't have to change it in the future if notebooks are added/moved.